### PR TITLE
Feature/av2 630/confusion while reviewing student answers

### PR DESCRIPTION
**Pull request by sasangachathumal**
Jira story - https://intersective.atlassian.net/browse/AV2-630
There are lots of UI changes and also some changes done to the code so please check Jira Story details and Comments.

### CODE REVIEW

File: src/app/questions/multiple/multiple.component.ts [hunk 1]
```diff
@@ -130,4 +132,13 @@ export class MultipleComponent implements ControlValueAccessor, OnInit {
     this.control.setValue(this.innerValue);
   }
 
+  // check question audience have more that one audience and is it includes reviewer as audience.
+  // then will identify it as a student and mentor answering in the same question and
+  // border need to add only for mentor section not for full question
+  private _checkReviewAudience() {
```
Comment (shawnm0705) [inline src/app/questions/multiple/multiple.component.ts:138]: same as above

File: src/app/questions/file/file.component.ts [hunk 2]
```diff
@@ -149,5 +152,13 @@ export class FileComponent implements ControlValueAccessor, OnInit {
     this.submission.answer = null;
     this.onChange('', null);
   }
+  // check question audience have more that one audience and is it includes reviewer as audience.
+  // then will identify it as a student and mentor answering in the same question and
+  // border need to add only for mentor section not for full question
+  private _checkReviewAudience() {
```
[the remark this answers is not in the document]
Comment (sasangachathumal) [reply src/app/questions/file/file.component.ts:158]: yes. got it. need to change that.

File: src/app/questions/file/file.component.ts [hunk 3]
```diff
@@ -49,6 +50,7 @@ export class FileComponent implements ControlValueAccessor, OnInit {
   comment: string;
   // validation errors array
   errors: Array<any> = [];
+  isReviewAudience: Boolean;
```
[the remark this answers is not in the document]
Comment (sasangachathumal) [reply src/app/questions/file/file.component.ts:53]: it's use for identify is this question have reviewer as audience.

File: src/app/questions/file/file.component.ts [hunk 4]
```diff
@@ -20,7 +20,8 @@ export class FileComponent implements ControlValueAccessor, OnInit {
     name: '',
     description: '',
     isRequired: false,
-    fileType: 'any'
+    fileType: 'any',
+    audience: []
```
Comment (shawnm0705) [inline src/app/questions/file/file.component.ts:24]: I think we are not using this variable anywhere

**Pull request merged**